### DISCUSSION TITLE
Brownie run scripts/deploy.py result in a ganache-cli permision error

### DISCUSSION THREAD

**Discussion by AndreeaCorinaO** [smartcontractkit/full-blockchain-solidity-course-py#149] [Q&A] [answered]
When I run in my bash brownie run scripts/deploy.py I get this error "PermissionError: [Errno 13] Permission denied: 'ganache-cli'".
I have checked the ganache-cli -- version and I have no such thing even if I have install it. I have go back and apparantly I can see neither node --version but is definetly there. I have give permission chmod 777 to the brownie_simple_storage directory but is not solving my problem.
Any ideas what exactly I could do to to permit my directory to see all the dependencies installed?

**Comment by PatrickAlphaC** [comment, marked as answer]
Could you let us know the exact output when you run:

1. `node --version`
2. `ganache-cli --version`
3. `brownie --version`


And additionally, does this help? 
https://stackoverflow.com/questions/48910876/error-eacces-permission-denied-access-usr-local-lib-node-modules
